### PR TITLE
Enabling chat for India team

### PR DESCRIPTION
**Pull request by khaled-zayed**
Edited the chat hours for US to be from 8-5 and enabled India from 7-8

### CODE REVIEW

File: Backend/appsettings.json [hunk 1]
```diff
@@ -195,22 +195,25 @@
...
             "OffHoursRegion": "USOffHours",
             "PublicHolidaysRegion": "USPublicHolidays"
+          },
+          {
+            "BusinessStartDay": 1, // Monday
+            "BuisnessEndDay": 5, //  Friday
+            "BusinessStartTimePST": "07:00", // 07:00 AM PST
+            "BusinessEndTimePST": "08:00", //  08:00 PM PST
+            "SupportTopics": [
+              "32542208", //Configuration and Management/Backup and Restore
```
Comment (puneetg1983) [inline Backend/appsettings.json]: Do you want to enable all these or just some specific ones?

File: Backend/appsettings.json [hunk 2]
```diff
@@ -175,7 +175,7 @@
           {
             "BusinessStartDay": 1, // Monday
             "BuisnessEndDay": 5, //  Friday
-            "BusinessStartTimePST": "07:00", // 07:00 AM PST
+            "BusinessStartTimePST": "08:00", // 07:00 AM PST
```
Comment (puneetg1983) [inline Backend/appsettings.json]: Correct the comment to say 8 am before this configuration starts getting messy :D

File: Backend/appsettings.json (hunk 1 above)
Comment (khaled-zayed) [reply Backend/appsettings.json]: All the config and management!

File: Backend/appsettings.json (hunk 2 above)
Comment (khaled-zayed) [reply Backend/appsettings.json]: Thanks!

**Pull request merged**